rpc/connection.go

```diff
@@ -454,14 +453,30 @@ func newConnectionWithTransportAndProtocols(handler ConnectionHandler,
 	return connection
 }
 
+func newConnectionWithTransportAndProtocols(handler ConnectionHandler,
+	transport ConnectionTransport, errorUnwrapper ErrorUnwrapper,
+	log LogOutput, opts ConnectionOpts) *Connection {
+	randBytes := make([]byte, 4)
+	rand.Read(randBytes)
+	connectionPrefix := fmt.Sprintf("CONN %s %x", handler.HandlerName(),
```

Comment:
Good point; will add!

Reply:
Actually the logrus one already has a random string from `connection_log.go`. But I'll move the unstructured on into that file as well to avoid confusion.